consumer/nf_management.go

```diff
@@ -81,21 +81,21 @@ var SendRegisterNFInstance = func(nrfUri, nfInstanceId string, profile models.Nf
 		} else if status == http.StatusCreated {
 			// NFRegister
 			resourceUri := res.Header.Get("Location")
-			resouceNrfUri = resourceUri[:strings.Index(resourceUri, "/nnrf-nfm/")]
+			resourceNrfUri = resourceUri[:strings.Index(resourceUri, "/nnrf-nfm/")]
 			retrieveNfInstanceId = resourceUri[strings.LastIndex(resourceUri, "/")+1:]
 			break
 		} else {
 			fmt.Println("handler returned wrong status code", status)
 			fmt.Println("NRF return wrong status code", status)
```

Comment:
Should we accordingly "fix" these logs as part of this PR? I would have expected that the `logger` to be used here instead of `fmt`
```suggestion
			logger.ConsumerLog.Errorf("handler returned wrong status code: %+v", status)
			logger.ConsumerLog.Errorf("NRF return wrong status code: %+v", status)
```
Moreover, would not a single log be enough?

Reply:
A single log line is kept.